wtxmgr/tx.go

```diff
@@ -3525,6 +3526,20 @@ func (s *Store) balanceFullScan(ns, addrmgrNs walletdb.ReadBucket, minConf int32
 			if spent {
 				return fmt.Errorf("spend credit found in unspent bucket")
 			}
+			ab.VotingAuthority += amtCredit
+			_, addrs, _, err := txscript.ExtractPkScriptAddrs(
+				txscript.DefaultScriptVersion, pkScript, s.chainParams)
+			if err != nil {
+				log.Warnf("ExtractPkScriptAddrs failed: %v", err)
```

Comment:
If you log an error, you have by definition handled the error.  Don't then return it to the caller.  Or don't log it.